src/main/java/racingcar/domain/car/Car.java

```diff
@@ -0,0 +1,44 @@
+package racingcar.domain.car;
+
+import racingcar.domain.car.strategy.RandomMoveStrategy;
+import racingcar.util.RandomUtils;
+
+public class Car {
+
+    private static final int START_POSITION = 0;
+    private static final int MIN_RANDOM_VALUE = 0;
+    private static final int MAX_RANDOM_VALUE = 9;
+
+    private final Name name;
+    private Position position;
+    private RandomMoveStrategy randomMoveStrategy;
+
+    public Car(final String name) {
+        this(new Name(name), new Position(START_POSITION));
+    }
+
+    public Car(final Name name, final Position position) {
+        this.name = name;
+        this.position = position;
+        this.randomMoveStrategy = new RandomMoveStrategy();
+    }
+
+    public boolean isSamePosition(final Position position) {
+        return this.position.equals(position);
+    }
+
+    public void goForwardRandomly() {
+        if (randomMoveStrategy.isMovable(
+                RandomUtils.generateRandomNumber(MIN_RANDOM_VALUE, MAX_RANDOM_VALUE))) {
```

Comment:
```suggestion
        if (moveStrategy.isMovable()) {
```

1. Random이라는 사실을 알아야 할 필요가 있을까요? Random이 아닌 규칙 기반으로 움직임이 결정된다면 어떻게 될까요?
2. 해당 메서드 내부에서 랜덤한 요소가 존재한다면 결국 정확한 테스트가 힘들지 않을까요?

위 리뷰에 대해 고민해보세요 :)